src/workspace/views/QuickLaneOverview.tsx

```diff
@@ -192,6 +202,13 @@ const QuickLaneOverview: FunctionComponent<QuickLaneOverviewProps> = ({ user })
 		fetchQuickLanes();
 	}, [fetchQuickLanes]);
 
+	useEffect(() => {
+		setQuery({
+			sort_column: query.sort_column || QUICK_LANE_DEFAULTS.sort_column,
+			sort_order: query.sort_order || QUICK_LANE_DEFAULTS.sort_order,
+		});
+	});
```

Comment:
A useEffect without a dependency array will execute on each render loop. This seems to cause an infinite render loop and essentially bricks the page. Did you mean to write this?
```suggestion
	}, [setQuery]);
```